docs/application/dotnet/guides/nui/animation.md

```diff
@@ -1,228 +1,195 @@
 
-<a name="readvalue"></a>
-### Reading Animated Values
+<div style="text-align:center;width:100%;"><img src="./media/easeout.svg" /></div>
 
-When a property is animatable, it can only be modified in the rendering thread. The value returned from a `get` property is the value used to render the previous frame.
+You can also specify the alpha function for each animation method:
 
-For example, `view.Position` has the position value at which the view was last rendered. Since defining a new position for `view.Position` is asynchronous, it does not immediately return the new value.
+```csharp
+// Animation in 0 ~ 500ms : speeds up
+animation.AnimateBy(view, "PositionX", 100, 0, 500, easeIn);
 
+// Animation in 500 ~ 1000ms : slows to a gradual stop
+animation.AnimateBy(view, "PositionX", 100, 500, 1000, easeOut);
 ```
-View view = new View();
-Window.Instance.Add(view); /// Initial position is 0,0,0
-view.Position = new Position(10, 10, 10);
 
-Position current = view.Position;
-Console.WriteLine("Current position: " + current.X + ", " + current.Y + ", " + current.Z);
-Console.WriteLine("...");
 
-/// Handle another event
+### Create Your Own Alpha Function
+You can create your own alpha function using `Bézier Curves`, by setting two control points, C1 and C2. The control points are for a Bézier curve whose end points are fixed at (0, 0) and (1, 1) respectively.
 
-current = view.Position;
-Console.WriteLine("Current position: " + current.X + ", " + current.Y + ", " + current.Z);
-```
+```csharp
+// Define two control points C1, C2
+var c1 = new Vector2(0.0f, 0.5f);
+var c2 = new Vector2(0.5f, 1.0f);
 
-The output of the code:
-
-```
-Current position: 0, 0, 0
-...
-Current position: 10, 10, 10
+var customAlphaFunction = new AlphaFunction(c1, c2);
 ```
+In the first graph shown in the following figure, the control points `C1(0, 0.5)` and `C2(0.5, 1)` make a curve that increases rapidly in the start and then slows down to the end. This means that the animation with this alpha function will have constant speed progress. You can see how the circle animates in the following graph:
 
-<a name="propsetting"></a>
-### Setting Properties during Animations
-
-When a property is being animated, the animation overrides all values set using other methods.
-
-The steps to execute in the render thread are:
-
-1.  Process the message and call the `SetPosition()` method.
-2.  Apply the animation and call the `SetPosition()` method.
-3.  Render the frame.
+<div style="text-align:center;width:100%;"><img src="./media/bezier.svg" /></div>
 
 <a name="sample"></a>
 ## Sample Animation
 
-You can use an `animation-hello-world.cs` sample to see in practice how the `AnimateBy()` and `AnimateTo()` methods, and alpha functions work.
+You can use an `animation-hello-world.cs` sample to practice and understand how the `AnimateBy()` and the `AnimateTo()` methods, and alpha functions work.
```

Comment:
Question. Isn't the sample name "animation-hello-world-tutorial.cs" according to line 194?
The sample name described here is different.